build.cake

```diff
@@ -1,7 +1,7 @@
 ///////////////////////////////////////////////////////////////////////////////
 // TOOLS
 ///////////////////////////////////////////////////////////////////////////////
-#tool "nuget:?package=JetBrains.ReSharper.CommandLineTools&version=2016.1.20160523.144749
+#tool "nuget:?package=JetBrains.ReSharper.CommandLineTools&version=2016.2.20160623.172854-eap5"
```

Comment:
@devlead I noticed that you changed this, and I was getting an error when I tried to resolve the version that I used.  Any ideas why it wasn't working?  The version 2016.1.20160523.144749 is on NuGet.